circle.yml

```diff
@@ -6,6 +6,6 @@ test:
   pre:
     - npm run eslint
   override:
-    - npm run istanbul
+    - npm test
```

Comment:
This override is probably unnecessary as I believe the Circle CI default is `npm test` - https://circleci.com/docs/language-nodejs/#testing